packages/driver/src/cypress/cy.ts

```diff
@@ -1205,6 +1205,12 @@ export default {
             // else just return ret
             return ret
           } catch (err) {
+            // If the runnable was marked as pending, this test was skipped
+            // go ahead and just return
+            if (runnable.isPending()) {
```

Comment:
What specifically is erroring in the above try block? Could we check is pending up there and return instead of relying on the try block?

Reply:
Mocha is actually throwing the error: https://github.com/mochajs/mocha/blob/ac43029d6a86150a48ccd59e50e89ca10c72a9c0/lib/runnable.js#L143

I don't believe there's a way we could catch it earlier than this as it's being triggered directly from the test being executed here: https://github.com/cypress-io/cypress/pull/19279/files#diff-231d74866a51e1411d172ab6ca0658f7fa712e9676f216ba797c544c47dcbf94R1144